site/source/docs/tools_reference/settings_reference.rst

```diff
@@ -2857,6 +2857,18 @@ then you can safely ignore this warning.
 
 Default value: false
 
+.. _single_file_binary_encode:
+
+SINGLE_FILE_BINARY_ENCODE
```

Comment:
How about calling this `SUPPORT_UTF8_EMBEDDING` to match the existing `SUPPORT_BASE64_EMBEDDING`?  

I do wish we could make this change without adding yet another setting, but I guess its not reasonable to make it unconditional.   Do you think we could make it unconditional at some point in the future?